### PR TITLE
net-misc/kea: bump, fix and drop old

### PR DESCRIPTION
**Pull request by expeditioneer**
Closes: https://bugs.gentoo.org/846620

### CODE REVIEW

File: net-misc/kea/kea-2.2.0.ebuild [hunk 1]
```diff
@@ -10,51 +10,69 @@ MY_P="${PN}-${MY_PV}"
...
-	EGIT_REPO_URI="https://github.com/isc-projects/kea.git"
+	EGIT_REPO_URI="https://gitlab.isc.org/isc-projects/kea.git"
 else
 	SRC_URI="ftp://ftp.isc.org/isc/kea/${MY_P}.tar.gz
 		ftp://ftp.isc.org/isc/kea/${MY_PV}/${MY_P}.tar.gz"
-	[[ "${PV}" == *_beta* ]] || [[ "${PV}" == *_rc* ]] || \
-	KEYWORDS="~amd64 ~arm64 ~x86"
+	# odd minor version = development release
+	if [[ $(( $(ver_cut 2) % 2 )) -ne 1 ]] ; then
+		[[ "${PV}" == *_beta* ]] || [[ "${PV}" == *_rc* ]] || KEYWORDS="~amd64 ~arm64 ~x86"
```
Comment (thesamesam) [inline net-misc/kea/kea-2.2.0.ebuild:25]: Just use a proper if statement instead and combine the ||s within one [[ ]].

File: net-misc/kea/kea-2.2.0.ebuild [hunk 2]
```diff
@@ -10,51 +10,69 @@ MY_P="${PN}-${MY_PV}"
...
+IUSE="debug doc mysql +openssl postgres +samples shell test"
 RESTRICT="!test? ( test )"
 
 COMMON_DEPEND="
 	dev-libs/boost:=
 	dev-libs/log4cplus
+	doc? (
+		$(python_gen_cond_dep '
+		  dev-python/sphinx[${PYTHON_USEDEP}]
+		  dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}]')
```
Comment (thesamesam) [inline net-misc/kea/kea-2.2.0.ebuild:40]:
```suggestion
		  dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}]
')
```

File: net-misc/kea/kea-2.2.0.ebuild [hunk 3]
```diff
@@ -10,51 +10,69 @@ MY_P="${PN}-${MY_PV}"
...
 
+pkg_setup() {
+	use shell && python-single-r1_pkg_setup
+}
+
 src_prepare() {
 	default
-	# Brand the version with Gentoo
+
+	cp "${FILESDIR}"/ax_gtest.m4 "${S}"/m4macros/ax_gtest.m4
```
Comment (thesamesam) [inline net-misc/kea/kea-2.2.0.ebuild:71]: || die if an external command may fail.

File: net-misc/kea/kea-2.2.0.ebuild (hunk 1 above)
Comment (expeditioneer) [reply net-misc/kea/kea-2.2.0.ebuild]: done

File: net-misc/kea/kea-2.2.0.ebuild (hunk 2 above)
Comment (expeditioneer) [reply net-misc/kea/kea-2.2.0.ebuild]: done

File: net-misc/kea/kea-2.2.0.ebuild (hunk 3 above)
Comment (expeditioneer) [reply net-misc/kea/kea-2.2.0.ebuild]: done

**Comment by expeditioneer** [comment]
@thesamesam thank you for having a look.

**Review by thesamesam — APPROVED** [review]

**Pull request closed without merging**